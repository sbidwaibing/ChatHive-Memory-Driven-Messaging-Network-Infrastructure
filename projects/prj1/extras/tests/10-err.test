+ @zdu test #tag
Test message
.
  x
.
 +
.
 + 	test
.
+ @user 
.
+   @user #topic
.
+ @user    room   topic
.
+ @user    room   #topic
.
 ?
.
? @user
.
  ? room
.
? test #tag1
.
? test #tag
