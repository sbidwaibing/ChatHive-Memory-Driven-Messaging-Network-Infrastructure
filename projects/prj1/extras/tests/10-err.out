BAD_COMMAND
BAD_USER
BAD_USER
BAD_ROOM
BAD_ROOM
BAD_TOPIC
NO_MSG
BAD_ROOM
BAD_ROOM
BAD_ROOM
BAD_TOPIC
@zdu test #tag
Test message
